src/sentry/api/endpoints/project_details.py

```diff
@@ -394,26 +403,30 @@ def get(self, request: Request, project: Project) -> Response:
 
         return Response(data)
 
+    @extend_schema(
+        operation_id="Update a Project",
+        parameters=[
+            GlobalParams.ORG_SLUG,
+            GlobalParams.PROJECT_SLUG,
+            GlobalParams.name("The name for the project."),
+            GlobalParams.slug("The slug for the project."),
+            ProjectParams.platform("The platform for the project."),
+            ProjectParams.IS_BOOKMARKED,
+        ],
+        request=ProjectAdminSerializer,
+        responses={
+            200: DetailedProjectSerializer,
+            403: RESPONSE_FORBIDDEN,
+            404: RESPONSE_NOT_FOUND,
+        },
+        examples=ProjectExamples.DETAILED_PROJECT,
+    )
     def put(self, request: Request, project) -> Response:
         """
-        Update a Project
-        ````````````````
-
-        Update various attributes and configurable settings for the given
-        project.  Only supplied values are updated.
-
-        :pparam string organization_slug: the slug of the organization the
-                                          project belongs to.
-        :pparam string project_slug: the slug of the project to update.
-        :param string name: the new name for the project.
-        :param string slug: the new slug for the project.
-        :param string platform: the new platform for the project.
-        :param boolean isBookmarked: in case this API call is invoked with a
-                                     user context this allows changing of
-                                     the bookmark flag.
-        :param int digestsMinDelay:
-        :param int digestsMaxDelay:
-        :auth: required
+        Update various attributes and configurable settings for the given project.
+
+        Note that soley having the **`project:read`** scope restricts updatable settings to
+        `isBookmarked` only.
```

Comment:
We have a weird permission scope check where we allow `project:read` to hit the endpoint, but restrict it from updating almost every field except `isBookmarked` and `isSubscribed`. AFAIK `isSubscribed` is irrelevant/useless so I excluded it. I'm not sure why this was allowed is the first place but I'm sure there was a reason so I'm afraid that removing `project:read` will break something in the FE